File: frontend/src/app/core/classes/orderLine.ts
import { Drink } from "./drink";
import {OrderStatus} from "./order-status";
import {User} from "./user";

export class OrderLine {
  orderLineId: string;
  drink: Drink;
  count: number;
  customer: User;
  server?: User;
  halen? : boolean;
  milk: number;
  sugar: number;
  orderStatus: OrderStatus;

}

File: backend/CoffeeAPI/Models/OrderLine.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace CoffeeAPI.Models
{
    public class OrderLine
    {
        public Guid OrderLineId { get; set; }

        [Required]
        public User Customer { get; set; }
        public User Server { get; set; }
        [Required]
        public Drink Drink { get; set; }
        public int Count { get; set; }
        public int Sugar { get; set; }
        public int Milk { get; set; }
        public OrderStatus OrderStatus { get; set; }
        public DateTime OrderTime { get; set; }
        public DateTime GetTime { get; set; }
    }
}
